docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -63,85 +76,99 @@ add_executable(ov_c_app main.c)
 
-Older OpenVINO packages:
+**With Inference Engine of previous versions**:
 
 @sphinxdirective
 
 .. tab:: Include dirs
 
   .. code-block:: sh
-
-    <openvino_root>/deployment_tools/inference_engine/include
-    <openvino_root>/deployment_tools/ngraph/include
+    
+    <INSTALL_DIR>/deployment_tools/inference_engine/include
+    <INSTALL_DIR>/deployment_tools/ngraph/include
 
 .. tab:: Path to libs
 
   .. code-block:: sh
 
-    <openvino_root>/deployment_tools/inference_engine/lib/intel64/Release
-    <openvino_root>/deployment_tools/ngraph/lib/
+    <INSTALL_DIR>/deployment_tools/inference_engine/lib/intel64/Release
+    <INSTALL_DIR>/deployment_tools/ngraph/lib/
 
 .. tab:: Shared libs
 
   .. code-block:: sh
 
-    # Unix systems
+    // UNIX systems
     inference_engine.so ngraph.so
 
-    # Windows OS
+    // Windows
     inference_engine.dll ngraph.dll
 
-.. tab:: (Win) .lib files
+.. tab:: (Windows) .lib files
 
   .. code-block:: sh
-
+  
     ngraph.lib
     inference_engine.lib
 
 @endsphinxdirective
 
-OpenVINO 2.0 package:
+**With OpenVINO Runtime 2022.1 (API 2.0)**:
 
 @sphinxdirective
 
 .. tab:: Include dirs
 
   .. code-block:: sh
 
-    <openvino_root>/runtime/include
+    <INSTALL_DIR>/runtime/include
 
 .. tab:: Path to libs
 
   .. code-block:: sh
 
-    <openvino_root>/runtime/lib/intel64/Release
+    <INSTALL_DIR>/runtime/lib/intel64/Release
 
 .. tab:: Shared libs
 
   .. code-block:: sh
 
-    # Unix systems
+    // UNIX systems
     openvino.so
 
-    # Windows OS
+    // Windows
     openvino.dll
 
-.. tab:: (Win) .lib files
+.. tab:: (Windows) .lib files
 
   .. code-block:: sh
 
     openvino.lib
 
 @endsphinxdirective
 
-## Deployment
+## Clearer Library Structure for Deployment
+
+OpenVINO 2022.1 has reorganized the libraries to make it easier for deployment. In previous versions, to perform deployment steps, you have to use several libraries. Now you can just use `openvino` or `openvino_c` based on your developing language plus necessary plugins to complete your task. For example, `openvino_intel_cpu_plugin` and `openvino_ir_frontend` plugins will enable you to load OpenVINO IRs and perform inference on CPU device.
 
-### Libraries reorganization
+Here you can find some detailed comparisons on library structure between OpenVINO 2022.1 and previous versions:
 
+* A single core library with all the functionalities (`openvino` for C++ Runtime, `openvino_c` for Inference Engine API C interface) is used in 2022.1, instead of the previous core libraries which contain `inference_engine`, `ngraph`, `inference_engine_transformations` and `inference_engine_lp_transformations`.
+* The optional `inference_engine_preproc` preprocessing library (if `InferenceEngine::PreProcessInfo::setColorFormat` or `InferenceEngine::PreProcessInfo::setResizeAlgorithm` is used) is deprecated in 2022.1. See more details on [Preprocessing capabilities of OpenVINO API 2.0](preprocessing.md).
```

Comment:
inference_engine_preproc  is renamed to openvino_gapi_preproc. But it's right, that it's not recommended for usage and OpenVINO API 2.0 should be used